config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -1537,7 +1537,7 @@
 # portal.error.email=
 
 # Email address used as the "from" address for any email sent by any Sakai tool.
-# If the value is not set, the tools will use no-reply@${serverName} or postmaster@${serverName}
+# If the value is not set, it will use smtpFrom@org.sakaiproject.email.api.EmailService, if not the tools will use no-reply@${serverName}
```

Comment:
```suggestion
# If the value is not set, it will use smtpFrom@org.sakaiproject.email.api.EmailService, if none of them have been set then the tools will use no-reply@${serverName}
```